examples/next-app/src/data/names.ts

```diff
@@ -0,0 +1,1010 @@
+export function randomAge(): number {
+  return Math.ceil(Math.random() * 99);
+}
+
+export function randomName(): string {
+  return Names[Math.floor(Math.random() * Names.length)];
+}
+
+export const Names = [
```

Comment:
Where does this come from? Also, what's it sorted by?

Reply:
This is just a randomly generated list (I think from ChatGPT)? I experimented with making an API call to randomly generate a user (e.g. https://randomuser.me/api), but it adds a little delay to the server side and client side mutations. I think it's more impactful to show how quickly the inserts happen without an async dependency like this.